great_expectations/render/view/templates/page.j2

```diff
@@ -160,7 +160,6 @@
       </div>
     </div>
     <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.15.0/umd/popper.min.js"></script>
```

Comment:
Are you sure we don't need popper? I thought it was required for tooltips. Does Taylor's `cooltip` avoid this dependency?

Reply:
Yep. It was required for tooltips when we were using bootstrap tooltips, but Taylor's cooltip implementation is pure CSS.